assets/js/lib/test-utils/factories/hosts.js

```diff
@@ -37,14 +37,69 @@ export const slesSubscriptionFactory = Factory.define(() => ({
   version: '15.3',
 }));
 
-export const saptuneStatusFactory = Factory.define(() => ({
-  package_version: faker.helpers.fake(
-    '{{number.int}}.{{number.int}}.{{number.int}}'
-  ),
-  configured_version: faker.number.int(),
-  tuning_state: saptuneTuningStateEnum(),
+const sapNoteID = () => faker.number.int({ min: 100000, max: 999999 });
+
+const sapNotesList = (count = 6) =>
+  Array(count)
```

Comment:
Let's use `lodash`, much more elegant
```
import { times } from 'lodash';
...
const sapNotesList = (count = 6) => times(count, sapNoteID)
```

Reply:
Aye this is nice, need to get used to lodash!